config/jetpack-cloud-development.json

```diff
@@ -16,7 +16,8 @@
 		"jetpack-cloud/backups-restore": true,
 		"jetpack-cloud/settings": true,
 		"jetpack-cloud/scan": true,
-		"jetpack-cloud/scan-history": true
+		"jetpack-cloud/scan-history": true,
+		"jetpack-cloud/on-demand-scan": true
```

Comment:
What's the idea behind this key?

Reply:
It makes the button Scan now button show up. So that it is easier to test.